bogo/src/main.rs

```diff
@@ -224,8 +226,9 @@ impl SelectedProvider {
             .as_deref()
         {
             None | Some("aws-lc-rs") => Self::AwsLcRs,
-            #[cfg(feature = "fips")]
-            Some("aws-lc-rs-fips") => Self::AwsLcRsFips,
+            // NOW REMOVED FROM THIS FORK:
+            // #[cfg(... "fips")]
+            // Some("aws-lc-rs-fips") => ...,
```

Comment:
minor nit:

```suggestion
            // Some("aws-lc-rs-fips") => ...
```

I think this should slightly improve consistency with other cases of partial commented-out code.